### ISSUE TITLE
E8b/E9b - Insufficient colour contrast/font-size on Event Grid Calendar (Monthly/Weekly)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#615]
Monthly and weekly event grid calendar has poor colour contrast (blue on blue) on all day events. In addition, font size is quite small.

![image](https://cloud.githubusercontent.com/assets/1927902/22348767/f93b8238-e3da-11e6-9c97-14bbdaeb1f9f.png)

Recommend:
- Improving the colour contrast to WCAG 2.0 Level AA conformance
- Increasing the font size of all the text.

We're putting this at a higher priority as it's something that may be a quick fix with a strong impact for users.

**Comment by ushenoyuog** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [P 35 Insufficient colour contrast/font-size on Event Grid](https://trello.com/c/hRGgbqNn/234-p-35-insufficient-colour-contrast-font-size-on-event-grid)

**Comment by ushenoyuog** [comment]
## Acceptance Criteria

1. - All Day must have contrast that meets WCAG 2.0 Level AA. Currently blue-on-blue fails to meet contrast requiremnets. 
2. - This must be fied on All Day events in both the weekly and monthly calendar grid. 
3. - If possible the font size must be increased a litle bit to have better readability.

**Comment by ushenoyuog** [comment]
Looks like this is done .. closing
